types.go

```diff
@@ -95,7 +96,10 @@ var authorType = graphql.NewObject(
 			},
 			"lastName": &graphql.Field{
 				Type: graphql.String,
-			},
+                        },
```

Comment:
This and below indentations are correct?

Reply:
Ehh they are not. I will fix that later